packages/gcp/changelog.yml

```
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "1.2.0"
+  changes:
+    - description: Add GCP Billing Metricset
+      type: enhancement
+      link: https://github.com/elastic/integrations/pull/XX
```

Adding correct PR link to changelog
```suggestion
      link: https://github.com/elastic/integrations/pull/2141
```